include/exec/system_context.hpp

```diff
@@ -82,17 +82,17 @@ namespace exec {
 
   } // namespace __detail
 
-  class system_scheduler;
-  class system_sender;
+  class parallel_scheduler;
+  class __parallel_sender;
   template <stdexec::sender _S, std::integral _Size, class _Fn>
-  class system_bulk_sender;
+  class __parallel_bulk_sender;
 
   /// Returns a scheduler that can add work to the underlying execution context.
-  auto get_system_scheduler() -> system_scheduler;
+  auto get_parallel_scheduler() -> parallel_scheduler;
```

Comment:
can you pls add back a (deprecated) `get_system_scheduler()` function?

Reply:
Done. thank you.